commerce_coordinator/apps/frontend_app_payment/urls.py

```diff
@@ -4,10 +4,15 @@
 
 from django.urls import path
 
-from commerce_coordinator.apps.frontend_app_payment.views import DraftPaymentCreateView, PaymentGetView
+from commerce_coordinator.apps.frontend_app_payment.views import (
+    DraftPaymentCreateView,
+    GetActiveOrderView,
+    PaymentGetView
+)
 
 app_name = 'frontend_app_payment'
 urlpatterns = [
     path('payment', PaymentGetView.as_view(), name='get_payment'),
     path('payment/draft', DraftPaymentCreateView.as_view(), name='creat_draft_payment'),
+    path('order/', GetActiveOrderView.as_view(), name='order'),
```

Comment:
In [spec URL](https://github.com/edx/commerce-coordinator-private/blob/main/docs/api-spec/theseus-api-spec.md#output-13) is different.

Reply:
updated to match spec